src/modules/store.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable max-classes-per-file */
+// local Storage
+export class Task {
```

Comment:
- [ ] Also, the edit task is not implemented. It's not hard to give suggestions if you can't describe where are you stuck. Please write which point of creating an edit task is stopping you.

Reply:
Hi @Rustamxon7 , thank you for your review. 
In my ui.js file from line 39 to 45, I tried to implement the editing function but it's not working properly. Could you kindly check and help me know where am going wrong?

Reply:
Hello @sharon-odhiambo, Sorry I was not able to reply in time to you. But looks like you solve the issue, right?

Reply:
Hi, yeah sure. I did. Thank you.
No worries at all.